tests/unit/task_system_test.c

```diff
@@ -349,6 +388,9 @@ exec_one_thread_use_lower_apis(void *arg)
    ASSERT_EQUAL(thread_cfg->active_threads_bitmask,
                 task_bitmask_before_register);
 
+   platform_default_log("active_threads_bitmask=0x%lx\n",
+                        thread_cfg->active_threads_bitmask);
```

Comment:
#494 changed all unit tests to use the `CTEST_LOG_INFO()` macro.  Any reason not to stick with that new pattern?

Reply:
@rosenhouse - This is left-over code that I did not sync-up with the interfaces of your change. Will revisit all new code and will conform to the new methods.